scripts/verify-structure.py

```
@@ -496,12 +644,61 @@ def check_custom_component_paths(plugin_dir: Path, plugin_data: dict) -> list[st
+                f"marketplace: {repr(market_value)}, "
+                f"plugin.json: {repr(plugin_value)} "
+                f"(plugin.json takes precedence)"
+            )
+
+    return warnings
+
+
+def check_plugin_manifest(
+    plugin_dir: Path,
+    marketplace_entry: dict | None = None,
+    strict_mode: bool = True
```

Ruff (0.14.4)

Boolean-typed positional argument in function definition

(FBT001)